### PR TITLE
Cleaning some code

### PR DESCRIPTION
**Pull request by igordcsouza**
## Description
Cleaning some code

## Motivation and Context
HACKTOBERFEST :)

## How Has This Been Tested?
rspec

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: lib/colorls/core.rb [hunk 1]
```diff
@@ -306,8 +301,8 @@ def options(path, content)
     def tree_traverse(path, prespace, indent)
       contents = init_contents(path)
       contents.each do |content|
-        icon = content == contents.last || Dir.exist?("#{path}/#{content}") ? ' └──' : ' ├──'
-        print tree_branch_preprint(prespace, indent, icon).colorize(@colors[:tree])
+        content == contents.last || Dir.exist?("#{path}/#{content}") ? ' └──' : ' ├──'
+        print tree_branch_preprint(prespace, indent, content).colorize(@colors[:tree])
```
Comment (athityakumar) [inline lib/colorls/core.rb]: Breaks `colorls -t`. Please revert these 2 lines.

Comment (igordcsouza) [reply lib/colorls/core.rb]: I'll try to make some tests for it... :)

**Comment by athityakumar** [comment]
@igordcsouza - Can you add the rubocop fixes in a separate PR and send a quick fix?

File: lib/colorls/core.rb [hunk 2]
```diff
@@ -184,7 +182,6 @@ def mode_info(stat)
           end
         end
       end
-      mode
```
Comment (GladOSkar) [inline lib/colorls/core.rb]: @athityakumar 

This change broke the Mode view

it now only displays 1's and 0's.

Please revert.

Comment (athityakumar) [reply lib/colorls/core.rb]: @GladOSkar - Can you add a small PR fixing the buggy parts of this PR?

**Pull request merged**